ui/pages/first-time-flow/first-time-flow-switch/first-time-flow-switch.component.js

```diff
@@ -34,7 +34,7 @@ export default class FirstTimeFlowSwitch extends PureComponent {
       return <Redirect to={{ pathname: DEFAULT_ROUTE }} />;
     }
 
-    if (seedPhraseBackedUp !== null) {
+    if (seedPhraseBackedUp) {
```

Comment:
From what I can tell, there is no case where seedphrase is used as either true | false | null. It seems to be only true | null.

Reply:
I'm hesitant to change this because we have a test to support `seedPhraseBackedUp` as false  in `ui/pages/first-time-flow/first-time-flow-switch/first-time-flow-switch.test.js`. See "redirects to end of flow route when seedPhraseBackedUp is false"

Reply:
Amazing, thank you for pointing this out. I will revert this.